cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,69 @@
-describe('', () => {
-  before(() => {
+/// <reference types="cypress" />
 
-  });
+const { faker } = require('@faker-js/faker');
 
-  it('', () => {
+describe('article', () => {
+  beforeEach(() => {
+    cy.visit('user/register');
+  });
 
+  it('should create an article', () => {
+    const randomNumber = Math.random().toString().slice(2, 10);
+    const user = {
+      username: faker.lorem.word() + randomNumber,
+      email: faker.internet.email(),
+      password: faker.internet.password()
+    };
+    const article = {
+      title: faker.lorem.word(),
+      description: faker.lorem.words(3),
+      body: faker.lorem.words(10)
+    };
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body)
+      .as('article');
+    cy.get('@article').should((resp) => {
+      expect(resp.status).to.eq(200);
+    });
   });
 
-  it('', () => {
+  it('should delete the created article', () => {
+    const randomNumber = Math.random().toString().slice(2, 10);
+    const user = {
+      username: faker.lorem.word() + randomNumber,
+      email: faker.internet.email(),
+      password: faker.internet.password()
+    };
+    const article = {
+      title: faker.lorem.word(),
+      description: faker.lorem.words(3),
+      body: faker.lorem.words(10)
+    };
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body)
+      .then((resp) => {
+        cy.getCookie('auth').then((token) => {
+          const authToken = token.value;
 
+          cy.request({
+            method: 'DELETE',
+            url: `/api/articles/${resp.body.article.slug}`,
+            body: {
+              article: {
+                title: resp.body.article.title,
+                description: resp.body.article.description,
+                body: resp.body.article.body,
+                tagList: []
+              }
+            },
+            headers: {
+              Authorization: `Token ${authToken}`
+            }
+          });
+        });
+      }).as('delete');
+    cy.get('@delete').should((resp) => {
+      expect(resp.status).to.eq(204);
```

Comment:
There is no need to set the alias just for one line of code. Use .then() in this case.
You need to check if the new article is visible on the profile page. Checking a response status is not enough.